forc-pkg/src/source/mod.rs

```diff
@@ -60,6 +60,16 @@ impl Default for IPFSNode {
     }
 }
 
+impl IPFSNode {
+    pub fn fuel() -> Self {
```

Comment:
```
/// Returns an IPFSNode configured to use the Fuel-operated IPFS gateway.
```

Reply:
done in [a834871](https://github.com/FuelLabs/sway/pull/7038/commits/a834871f5a097a293ea5bc053229d0ab8eccf566)